README.md

```diff
@@ -160,6 +160,9 @@ function getAllAddons({ categoryId }: GetAllAddonsParams = {}) {
   the keyboard. If it's something you think is unfixable then use
   `$FLOW_IGNORE` instead. Please explain your rationale in the comment and link
   to a GitHub issue if possible.
+* If you're stumped on why some Flow annotations aren't working, try using
```

Comment:
Ooh, I just remembered that unlike with `npm`, you can pass args to `yarn` shortcuts more easily. I can add a `yarn flow` shortcut which will allow things like `yarn flow type-at-pos ...`.